src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoParticipantData.java

```diff
@@ -0,0 +1,93 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.sagebionetworks.bridge.models.ParticipantData;
+
+import java.util.Objects;
+
+/**
+ *
+ */
+@DynamoDBTable(tableName = "ParticipantData")
+public class DynamoParticipantData implements ParticipantData {
+
+    private String userId;
```

Comment:
Purely for reasons of convention, I would use healthCode as the user identifier here, not userId.

Reply:
I will change it! Are there any examples I should look at to help me understand the usage or naming convention of healthCode?

Reply:
Clarifying question: Just for the ParticipantData object or for the entire API? The API should definitely use userId, since it's a pain in the neck for callers (such as the workers) to use healthCode.

Using the healthCode in DynamoDB means the service would need to make an extra database call to fetch the account and get the healthCode. Which might be fine, but it's an extra step that we would need to take. (Or it might not be fine. Evidence suggests that Bridge Server has been getting slower in the last 6 months or so.)